src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -1192,6 +1192,7 @@ def load_arguments(self, _):
     for scope in ['network vnet subnet list', 'network vnet peering list']:
         with self.argument_context(scope) as c:
             c.argument('ids', deprecate_info=c.deprecate(hide=True, expiration='2.1.0'))
+            c.argument('virtual_network_name', id_part=None)
```

Comment:
When you make id_part=None, --ids will never show up, right? In this way, you just remove --ids for this command, not waiting for 2.1.0 to deprecate?

Reply:
Currently, user cannot see the `--ids` as well since it has been hide for one year. That's the reason that it's a breaking change.